### ISSUE TITLE
Check tutorials in CI

### ISSUE THREAD

**Issue by samestep** [penrose/penrose#859]
#858 updates the tutorials so they are (hopefully) correct as of 2022-01-17 (modulo `inRange` being broken I think), but we currently have no mechanism to ensure that they stay up to date in the future. I propose that we do the following:

1. Move everything from https://github.com/penrose/tutorials into this repository and archive the former.
2. Make Penrose deterministic: #731
3. Replace the handmade screenshots in the tutorials with autogenerated SVGs from the solutions.
4. Add a GitHub Actions check to ensure that those SVGs are up to date.
5. Also parse the Markdown for the wiki pages to ensure they line up with the sample solutions.

**Comment by wodeni** [comment]
Good idea! BTW since the tutorial is MDX now. Maybe we can use `@penrose/components` in the tutorial directly? This is a good opportunity for dogfooding our React components.

**Comment by samestep** [comment]
Oh interesting! I don't really know what that means tbh but it sounds cool, will look into it.
